files/ib_perftest.yml

```diff
@@ -0,0 +1,168 @@
+                         description: 'Do the perftest using ib_write*'
+                 - choice:
+                         name: do_mpitest
+                         choices:
+                                 - True
+                                 - False
+                         default: 'True'
+                         description: 'Do the MPI perftests'
+                 - string:
+                         name: latency_threshold
+                         default: '3.0'
+                         description: 'Threshold above which Latency test is failed'
+                 - string:
+                         name: bw_threshold
+                         default: '45.0'
+                         description: 'Threshold below which Bandwidth test is failed'
+                 - string:
+                         name: bibw_threshold
+                         default: '45.0'
+                         description: 'Threshold below which Bidirectional Bandwidth test is failed'
+                 - string:
+                         name: mpilat_threshold
+                         default: '3.0'
+                         description: 'Threshold above which MPI Latency test is failed'
+                 - string:
+                         name: mpibw_threshold
+                         default: '1700'
+                         description: 'Threshold below which MPI Bandwidth test is failed'
+                 - string:
+                         name: mpibibw_threshold
+                         default: '1700'
+                         description: 'Threshold below which MPI Bidirectional Bandwidth test is failed'
+                 - string:
+                         name: git_branch
+                         default: 'production'
+                         description: 'Branch name of the ohpc install recipe'
+         builders:
+                 - shell: |
+                        #!/bin/bash
+                        set -ex
+                        if [ -d ${WORKSPACE}/mr-provisioner-client ]; then
+                            rm -rf ${WORKSPACE}/mr-provisioner-client
+                        fi
+                        git clone https://github.com/Linaro/mr-provisioner-client.git ${WORKSPACE}/mr-provisioner-client
+
+                        mr_provisioner_url='http://10.40.0.11:5000'
+                        mr_provisioner_token=$(cat "/home/${NODE_NAME}/mrp_token")
+
+                        host_machine_ip=$( ${WORKSPACE}/mr-provisioner-client/mrp_client.py getip ${host_machine} --mrp-url=${mr_provisioner_url} --mrp-token=${mr_provisioner_token})
+                        remote_machine_ip=$( ${WORKSPACE}/mr-provisioner-client/mrp_client.py getip ${remote_machine} --mrp-url=${mr_provisioner_url} --mrp-token=${mr_provisioner_token})
+
+                        # Get the XXX in 10.40.24.XXX to make 192.168.0.XXX
+                        IFS="." read -ra REMOTE <<< "$remote_machine_ip"
+                        remote_ib_ip="192.168.0.${REMOTE[-1]}"
```

Comment:
I'll go for 100.*, but note that this range will not conflict with anything since you need to be logged into a lab machine to have access to it (I.e. use their routing tables).

Reply:
One could create a bridge (like sshuttle) to reach it in a more convenient way from VPN.